_assets/ci/Jenkinsfile.tests

```diff
@@ -0,0 +1,69 @@
+pipeline {
```

Comment:
Doesn't it need to be used somewhere?

Reply:
What do you mean?

Reply:
I am just wondering where and how this Jenkinsfile is used.

Reply:
https://ci.status.im/job/status-go/job/prs/job/tests/view/change-requests/ - here

Reply:
And you can see it below, as `jenkins/prs/tests`.

Reply:
Oh ok, configured directly in Jenkins. All clear.